fvtest/threadextendedtest/processTimeTest.cpp

```diff
@@ -301,14 +278,14 @@ appThread(void *arg)
 	return 0;
 }
 
-#define NUM_APP_THREAD	50
-#define NUM_SYS_THREAD	10
+#define NUM_APP_THREAD 50
+#define NUM_SYS_THREAD 10
 
 /**
  * Compare the values of omrthread_get_cpu_time and omrthread_get_self_cpu_time
  * and the monotonicity of the omrthread_get_cpu_time values
  */
-TEST(ThreadExtendedTest, TestThreadCpuTime)
+TEST(ThreadExtendedTest, DISABLED_TestThreadCpuTime)
```

Comment:
Yes. Disabling until it's run time is reduced.

There are two new CpuTimeTest unit tests for coverage on `omrthread_get_jvm_cpu_usage_info`.

Both TestThreadCpuTime and CpuTimeTest check for monotonic cpu time growth across the entire process. The difference is TestThreadCpuTime brings up multiple threads to burn the CPU, the CpuTimeTest is single-threaded.